src/runtime/audience-action-flow.js

```diff
@@ -293,6 +298,15 @@ export class AudienceActionFlow {
     // @TODO(justinchou): execute callback with setOnInterventionComplete
     // then check for success
     const gaLoggingSuccess = this.logSurveyDataToGoogleAnalytics(request);
+    if (!gaLoggingSuccess) {
```

Comment:
Currently, this is only triggered with GA/gTag eligibility check fails. I believe we need to modify https://github.com/subscriptions-project/swg-js/blob/main/src/runtime/client-event-manager.js#L135-L144 to propagate up any errors so that this can also account for any errors that happen during the logging execution. Can you confirm this is correct?

Reply:
I feel like the eligibility check can be sufficient for the experiment given that we'll eventually defer logging to publisher's callback.